cookbooks/bcpc-hadoop/templates/default/hdp_core-site.xml.erb

```diff
@@ -120,4 +120,51 @@
     <name>hadoop.user.group.static.mapping.overrides</name>
     <value>hdfs=hadoop,hdfs;yarn=mapred,hadoop;</value>
   </property>
+  <% if node[:bcpc][:hadoop][:hdfs][:ldap][:integration] == true %>
+   <!-- HDFS LDAP Mapping configuration -->
+  <property>
+    <name>hadoop.security.group.mapping</name>
+    <value>org.apache.hadoop.security.LdapGroupsMapping</value>
+  </property>
+
+  <property>
+    <name>hadoop.security.group.mapping.ldap.bind.user</name>
+    <value><%= node[:bcpc][:hadoop][:hdfs][:ldap][:user] %></value>
+  </property>
+
+  <property>
+    <name>hadoop.security.group.mapping.ldap.bind.password.file</name>
```

Comment:
Can this be a `-D` option for the daemons which need it but so that users do not need read privileges?